src/features/slackapi/slackapi-utils.ts

```diff
@@ -187,7 +197,7 @@ Have a great week!
     const { weekStartDate, weekEndDate } = TimeUtilities.lastWeekDateProvider();
     const { yesterday, today } = previousWorkDays;
 
-    const messageResults: WeeklyMessageResult[] = [];
+    const messageResults: any[] = [];
```

Comment:
Check if this change of type from WeeklyMessageResult to any is necessary. We want to avoid using 'any' where ever possible.

Reply:
converted back to WeeklyMessageResult[]